helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -422,6 +422,7 @@ agent:
     # create a service for the agents with a local internalTrafficPolicy
     # so that agent pods can be discovered via dns etc
     enabled: true
+    trafficDistribution: ""
```

Comment:
I think this config would be a no-op for our agent k8s service since it has like strict traffic distribution guarantee already with the hardcoded `internalTrafficPolicy: Local` in there

Reply:
You're right. Let me remove this from the agent